src/server/wesnothd/server.cpp

```diff
@@ -1372,16 +1457,17 @@ void server::handle_create_game(player_iterator player, simple_wml::node& create
 	const std::string game_name = create_game["name"].to_string();
 	const std::string game_password = create_game["password"].to_string();
 	const std::string initial_bans = create_game["ignored"].to_string();
-	const bool is_queue_game = create_game["queue_game"].to_bool();
+	const cssv::QUEUE_TYPE queue_type = static_cast<cssv::QUEUE_TYPE>(create_game["queue_type"].to_int());
```

Comment:
enums should be stored as strings not as their numeric representation, i don't think we store enums as int anywhere else.

Reply:
it has multiple avantages, for example its more robust against future changes, lets say we add new "types" (tbh i didnt really read the code to know exactly what it does) and remove other types later, we'd end up with a messy api. (lets say the type with index 1 is removed then we'd suddenly have possible values 0,2,3 or something.

Also it's eaiser to read in case someone tries to read the wml data somehow.

Since the wesnothd communication is not a public/stable api, it's not _that_ bad here, i still think it makes sense to follow these best practices.

Reply:
Updated.